e2e/pages/vscode.pages.ts

```diff
@@ -128,56 +129,144 @@ class VSCode {
 
   /**
    * Iterates through all frames and returns the
-   * server status panel frame .
+   * left panel frame for further interactions.
    */
-  public async getServerStatusIframe(): Promise<FrameLocator | null> {
+  public async getLeftIframe(): Promise<FrameLocator | null> {
```

Comment:
This method was no longer used and was replaced by getServerStatusIframe() method .
https://github.com/konveyor/kai-ci/blob/main/e2e/pages/vscode.pages.ts#L133

Reply:
Solved in 8b9dc7a54e22957f6ca45afd3be04c0638867e11.